duckduckgo/iOS
# use special serp experiment variants

Task/Issue URL: https://app.asana.com/0/392891325557410/1200628097107837
Tech Design URL:
CC: @moollaza

**Description**:

Allocates new SERP variants for upcoming experiment.

**Steps to test this PR**:

1. Launch the app, ensure assigned to one of the variants
1. Fresh install until you get the other variant



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
